src/Specification/SpecificationParser.php

```diff
@@ -332,7 +332,7 @@ private function getProperty(string $propertyName, $property, bool $isRequest, a
         /** @var string|int|float|bool|null $schemaDefaultValue */
         $schemaDefaultValue = $itemProperty->default;
 
-        if ($schemaDefaultValue !== null && $isScalar && $scalarTypeId !== null) {
```

Comment:
It looks like a duplicate check. It doesn't work with it to write tests for mutations. Can I change it?